server/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const path = require('path');
+const db = require('./db/connection');
+const PORT = process.env.HOST_PORT || 8080;
+const app = express();
+const { seed } = require('../seed');
+const morgan = require('morgan');
+
+// for customizing env variables, .env needs to stay in root directory
+require('dotenv').config();
```

Comment:
Like it, but probably only do this in `NODE_ENV=development`, when you deploy to `Heroku`, there will be no `.env` file.